azkaban-common/src/main/java/azkaban/imagemgmt/dto/HPFlowDTO.java

```diff
@@ -0,0 +1,43 @@
+package azkaban.imagemgmt.dto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.validation.constraints.NotBlank;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+
+/**
+ * This is Add HP Flow class for updating high priority flow metadata.
+ */
+public class HPFlowDTO extends BaseDTO {
+  // CSV of flow ids.
+  @JsonProperty("flowIds")
+  @NotBlank(message = "flowList cannot be empty.")
+  private String flowIds;
+
+  public void setFlowIds(final String flowIds) {
+    // Remove all whitespaces
+    this.flowIds = flowIds.replaceAll("\\s", "");
+  }
+
+  public String getFlowIds() {
+    return this.flowIds;
+  }
+
+  /**
+   * Converts the CSV into a list of flow IDs.
+   * @return list of flow IDs.
+   */
+  public List<String> getFlowIdList() {
+    if (this.flowIds.isEmpty()) {
```

Comment:
How is this possible without an error from the Not Blank annotation?

Reply:
Its a sanity check. If you think its useless, let me know.

Reply:
I think it would be better not to validate twice, but that is just my personal preference.